boot-qemu.py

```diff
@@ -72,21 +72,22 @@ def __init__(self):
         self._ram = '512m'
 
     def _find_dtb(self):
-        if not self._dtb:
-            raise RuntimeError('No dtb set?')
+        if not self._dtbs:
```

Comment:
Is an empty list a falsey value?

Reply:
Yes:

```
>>> not []
True
```

https://docs.python.org/3/library/stdtypes.html#truth-value-testing